data/datasets.md

```diff
@@ -141,3 +141,111 @@ To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the follo
 ### Datasets in dataset_definitions.yml
 * `VOC2007_detection` used for evaluation models on VOC2007 dataset for object detection task. Background label + label map with 20 object categories are used. (model examples: [`mobilenet-ssd`](../models/public/mobilenet-ssd/README.md), [`ssd300`](../models/public/ssd300/README.md))
 * `VOC2007_detection_no_bkgr` used for evaluation models on VOC2007 dataset for object detection tasks. Label map with 20 object categories is used.(model examples: [`yolo-v1-tiny-tf`](../models/public/yolo-v1-tiny-tf/README.md))
+
+## [PASCAL-S](http://cbs.ic.gatech.edu/salobj/)
+
+### How download dataset
+
+To download PASCAL-S dataset, you need to follow the steps below:
+1. Go to the [The Secrets of Salient Object Segmentation](http://cbs.ic.gatech.edu/salobj/) website
+2. Go to the `DOWNLOAD` section
+3. Select [`DATASET&CODE`](http://cbs.ic.gatech.edu/salobj/download/salObj.zip) and download archive
+4. Unpack archive
+
+### Files layout
+
+To use this dataset with OMZ tools, make sure `<DATASET_DIR>` contains the following:
+
+* `PASCAL-S` - directory containing images and salient region masks subdirectories
+    * `image` - directory containing the PASCAL-S images from the directory `datasets/imgs/pascal` in the unpacked archive
+    * `mask` - directory containing the PASCAL-S salient region masks from the directory `datasets/masks/pascal` in the unpacked archive
+
+### Datasets in dataset_definitions.yml
+* `PASCAL-S` used for evaluation models on PASCAL-S dataset for salient object detection task. (model examples: [`f3net`](../models/public/f3net/README.md))
+
+## [CoNLL2003 Named Entity Recognition](https://www.aclweb.org/anthology/W03-0419/)
+
+### How download dataset
+
+To download CoNLL2003 dataset, you need to follow the steps below:
+1. Go to the [CoNLL 2003 (English) Dataset](https://deepai.org/dataset/conll-2003-english) page in the [DeepAI Datasets](https://deepai.org/datasets) website
+2. Select [`DOWNLOAD`](https://data.deepai.org/conll2003.zip) and download archive
```

Comment:
Are you tried to convert it?
I used another source for downloading data, so possibly some issues in convertor can be